extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -142,14 +142,21 @@ export const nameMustNotBeEmpty = localize('nameMustNotBeEmpty', "Name must not
 export const selectPublishOption = localize('selectPublishOption', "Select where to publish the project to");
 export const publishToExistingServer = localize('publishToExistingServer', "Publish to existing server");
 export const publishToDockerContainer = localize('publishToDockerContainer', "Publish to new server in a container");
+export const publishToNewAzureServer = localize('publishToNewAzureServer', "Publish to new SQL DB logical server");
 export const enterPortNumber = localize('enterPortNumber', "Enter SQL server port number or press enter to use the default value");
 export const serverPortNumber = localize('serverPortNumber', "SQL server port number");
+export const azureServerName = localize('azureServerName', "SQL server name");
 export const serverPassword = localize('serverPassword', "SQL Server admin password");
```

Comment:
SQL server admin password (server the entity, not the brand)

Reply:
these are fixed in other branches. will merge soon